src/communication/communication.service.ts

```diff
@@ -67,4 +68,21 @@ export class CommunicationService {
 
     return true;
   }
+
+  handleError(e): WebsocketReturnType {
+    if (settings.env !== 'test') console.error(e);
```

Comment:
here what's mean about env to test?

Reply:
when running unit or e2e tests, `settings.env` is equal to `"test"` https://github.com/Tonomy-Foundation/Tonomy-Communication/blob/master/src/settings.ts#L5

I turned off the logging so we can have a clean test suit log.